support-lambdas/it-test-runner/cfn.yaml

```diff
@@ -40,7 +40,7 @@ Resources:
         RerunTests:
           Type: Schedule
           Properties:
-            Schedule: 'rate(1 hour)'
+            Schedule: 'rate(1 day)'
```

Comment:
"day" is from https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html